sdk/tests/conformance2/textures/misc/tex-srgb-mipmap.html

```diff
@@ -159,7 +157,34 @@
     }
 }
 
+function generateMipmap_immutableTexture()
+{
+    description("Generate mipmaps for immutable texture.");
```

Comment:
This overwrites the description from the first test. Please change the test to call description() once, describing all of the tests in the file at a high level, and then use debug() here and throughout to describe the individual tests.